### ISSUE TITLE
Obtaining `"{nodes,edges}_file"` for a given population.

### ISSUE THREAD

**Issue by 1uc** [BlueBrain/libsonata#218]
We're using `libsonata` to parse a SONATA circuit config file. The information we'd like to get is the morphology directory, the `"nodes_file"` and `"edges_file"` for a given population.

I was hoping for an API similar to:
```
circuit_config = libsonata.CircuitConfig.from_file(config_file)
node_prop = config.node_population_properties(population="NodeA")
nodes_file = node_prop.nodes_file # or something similar
morphology_dir = node_prop.morphology_dir # This exists and is very convenient.
```
I was unable to find something like this in the documentation. Is this already possible? (Does this even make sense?)

Currently, we're getting the information my manually crawling the extended JSON.

**Comment by 1uc** [comment]
That's indeed the other place I looked for this functionality.

The downside of needing to open the file is that it must exist. While this is guaranteed in our immediate usecase (SpatialIndex). It might be nice to be able to use the circuit config file when creating the circuit, i.e. in TouchDetector or a little bit later in the pipeline, where the `edges_file` is created.

Second thought: if you add it to `Population` and you ever want to create a `Population` not by loading it from disk, e.g., because all the data already resides in memory, the value of `filename` would be something brittle, like `None`.

**Comment by mgeplf** [comment]
> The downside of needing to open the file is that it must exist.

Yeah, I don't particularly like that idea.  Perhaps both `getting the paths` ways could exist. But for now, I can see the utility in having the `CircuitConfig` having the path available - I'll whip up a PR.

**Comment by mgeplf** [comment]
@1uc are you happy w/ this? Can I close the issue?
